src/components/Review.jsx

```diff
@@ -0,0 +1,37 @@
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react"
+
+
+
+const Review = () => {
+
+    const [isLoading, setIsLoading] = useState(true);
+    const [review, setReview] = useState({});
+    const { review_id } = useParams()
+
+    useEffect(() => {
+        fetch(`https://nc-games-portfolio-project.onrender.com/api/reviews/${review_id}`)

```

Comment:
This logic should be extracted out as a function and placed in a file named api.js